gtsam/hybrid/HybridFactorGraph.h

```diff
@@ -204,45 +288,50 @@ class HybridFactorGraph {
   HybridFactorGraph linearize(const Values& continuousValues) const;
 
   /**
-   * Utility for retrieving the internal DC factor graph
-   * @return the member variable dcGraph_
+   * @return true if all internal graphs of `this` are equal to those of
+   * `other`
    */
-  const DCFactorGraph& dcGraph() const;
+  bool equals(const HybridFactorGraph& other, double tol = 1e-9) const;
 
-  /**
-   * @return true if all internal graphs are empty
-   */
-  bool empty() const;
+  /// The total number of factors in the nonlinear factor graph.
+  size_t nrNonlinearFactors() const { return nonlinearGraph_.size(); }
 
-  /**
-   * @return true if all internal graphs of `this` are equal to those of `other`
-   */
-  bool equals(const HybridFactorGraph& other, double tol = 1e-9) const;
+  /// The total number of factors in the discrete factor graph.
+  size_t nrDiscreteFactors() const { return discreteGraph_.size(); }
 
-  /**
-   * @return the total number of factors across all internal graphs
-   */
-  size_t size() const;
+  /// The total number of factors in the DC factor graph.
+  size_t nrDcFactors() const { return dcGraph_.size(); }
 
-  /**
-   * @return the total number of factors in the nonlinear factor graph
-   */
-  size_t size_nonlinear() const;
+  /// The total number of factors in the Gaussian factor graph.
+  size_t nrDaussianFactors() const { return gaussianGraph_.size(); }
```

Comment:
Typo?  Should be`nrGaussianFactors` instead of `nrDaussianFactors`.

Reply:
Indeed!